ebmbot/job_configs.py

```diff
@@ -237,15 +236,15 @@
         "python_file": "generate_report.py",
         "jobs": {
             "generate_pipeline_report": {
-                "python_function": "main project_num=12 statuses=['in progress', 'blocked']",
+                "python_function": "main",
                 "run_args_template": "",
                 "report_stdout": True,
                 "report_format": "blocks",
             }
         },
         "slack": [
             {
-                "command": "pipeline report",
+                "command": "pipeline report [project_num] [statuses]",
```

Comment:
I'm not sure this is what you want - this means you have to call the commad with
```@bennett_bot teampipeline pipeline report 12 ["In Progress", "Blocked"]```

If you always want the same report number and statuses, defaults in the `generate_report` script would work

Reply:
Thank you for the explanation. You're right, it wasn't what I wanted. So, I've made some updates and made it so that it will be easy to move the data team over to using it too, once I know it works in production.